components/resizable-iframe/index.js

```diff
@@ -0,0 +1,193 @@
+/**
+ * Imported from Calypso, with some lint fixes and gutenburg specific changes.
+ */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import debugFactory from 'debug';
+import { omit } from 'lodash';
+import uuid from 'uuid/v4';
+
+/**
+ * Globals
+ */
+const debug = debugFactory( 'gutenburg:resizable-iframe' ),
+	noop = () => {};
+
+export default React.createClass( {
+	displayName: 'ResizableIframe',
+
+	propTypes: {
+		src: React.PropTypes.string,
+		width: React.PropTypes.oneOfType( [
+			React.PropTypes.string,
+			React.PropTypes.number
+		] ),
+		height: React.PropTypes.oneOfType( [
+			React.PropTypes.string,
+			React.PropTypes.number
+		] ),
+		onLoad: React.PropTypes.func,
+		onResize: React.PropTypes.func,
+		title: React.PropTypes.string
+	},
+
+	getInitialState: function() {
+		return { width: 0, height: 0 };
+	},
+
+	getDefaultProps: function() {
+		return {
+			onLoad: noop,
+			onResize: noop,
+			title: uuid()
```

Comment:
Maybe also cc @afercia on this one, in case I'm misspeaking. The guidelines do seem to strongly encourage that "no titles are repeated". We could include the Tweet's URL in the title as added uniqueness, but I don't sense this is valuable for screen reader users. The oEmbed response does include _some_ details, like Tweet author and the raw `html` (from which the tweet's content could be parsed, though not easily).